databook/background/experimental-setup.md

```diff
@@ -1,11 +1,11 @@
 # Behavioral apparatus
 
 Neurons in the visual system often have spatially selective receptive fields,
```

Comment:
This is the wrong motivation here. (this was added without review last year). Head fixation is motivated by wanting to record activity, not by how we show visual stimuli to the mice. I'll make an issue to revisit this section.